### PR TITLE
Change docs to launch the first stable version

### PR DESCRIPTION
**Pull request by Klynger**
#### What problem is this solving?

As the title says

#### How should this be manually tested?

[Workspace](https://quickview--storecomponents.myvtex.com/)

Also resolves https://github.com/vtex-apps/store-discussion/issues/127

#### Checklist/Reminders

- [X] Updated `README.md`.
- [ ] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes

Depends on https://github.com/vtex-apps/store-link/pull/1

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -23,35 +23,107 @@ WIP
```
Comment (victorhmp) [inline docs/README.md]: This is actually for the line above:

2. Now you can use the two blocks exported by `vtex.modal-layout`. Notice that you need to configure your own `modal-layout` and pass it as a child of `modal-trigger`.

Comment (victorhmp) [reply docs/README.md]: Also, for line 14, add the `"dependencies"` part to it:

```json
{
  "dependencies": {
      "vtex.modal-layout": "0.x"
  }
}
```

File: docs/README.md [hunk 2]
```diff
@@ -23,35 +23,107 @@ WIP
...
+  "product-price#quickview-price": {
+    "props": {
+      "showInstallments": true,
+      "showSavings": true
+    }
+  },
+  "link.product#quickview": {
+    "props": {
+      "href": "/{slug}/p",
+      "label": "Mais detalhes >",
```
Comment (victorhmp) [inline docs/README.md]: Might be better to have English text here

File: docs/README.md [hunk 3]
```diff
@@ -23,35 +23,107 @@ WIP
...
+      "product-bookmark",
+      "product-summary-specification-badges",
+      "modal-trigger#quickview"
+    ],
+    "props": {
+      "blockClass": "shelf-summary"
+    }
+  },
 }
 ```
```
Comment (victorhmp) [inline docs/README.md]: This is for the section below:
Kind of a nitpick, but sort the CSS handles list.
If you don't have this VS Code extension yet, should make it pretty easy: `tyriar.sort-lines`

**Comment by Klynger** [comment]
Solves https://github.com/vtex-apps/store-discussion/issues/114

**Pull request merged**